core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/SequencingParameters.java

```diff
@@ -3,45 +3,150 @@
 import java.io.Serializable;
 import java.util.Date;
 
-import javax.xml.xpath.XPathExpressionException;
-
-import org.w3c.dom.Document;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
 
 import com.eaglegenomics.simlims.core.User;
 
-public interface SequencingParameters extends Comparable<SequencingParameters>, Serializable {
+import uk.ac.bbsrc.tgac.miso.core.data.impl.PlatformImpl;
+import uk.ac.bbsrc.tgac.miso.core.data.impl.UserImpl;
+
+@Entity
+@Table(name = "SequencingParameters")
+
+public class SequencingParameters implements Comparable<SequencingParameters>, Serializable
+{
+
+  private static final long serialVersionUID = 1L;
+
+  @Enumerated(EnumType.STRING)
+  private IlluminaChemistry chemistry;
+
+  @ManyToOne(targetEntity = UserImpl.class, fetch = FetchType.LAZY)
+  @JoinColumn(name = "createdBy", nullable = false)
+  private User createdBy;
+
+  @Column(nullable = false)
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date creationDate;
+
+  @Column(nullable = false)
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date lastUpdated;
+  @Column(nullable = false)
+  private String name;
+
+  private boolean paired;
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long parametersId;
+
+  @ManyToOne(targetEntity = PlatformImpl.class)
+  @JoinColumn(name = "platformId")
+  private Platform platform;
+
+  private int readLength;
+
+  @ManyToOne(targetEntity = UserImpl.class, fetch = FetchType.LAZY)
+  @JoinColumn(name = "updatedBy", nullable = false)
+  private User updatedBy;
+
+  @Override
+  public int compareTo(SequencingParameters o) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override "equals(Object obj)" to comply with the contract of the "compareTo(T o)" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1210)